### ISSUE TITLE
Test File credit

### ISSUE THREAD

**Issue by kengdoj** [Section508Coordinators/baselinealignment#63]
If we reuse test files from outside sources (like TT), we should give credit where it's due. 

Can this be part of the test file or test case template?

**Comment by drewnielson** [comment]
Yes, I'll update the testcases.html and the _testcase-template.md file to add a citation/reference placeholder and a new field for authors to drop in a reference.
